app/coffee/WebsocketLoadBalancer.coffee

```diff
@@ -93,11 +100,9 @@ module.exports = WebsocketLoadBalancer =
 				Async.eachLimit clientList
```

Comment:
@ShaneKilkelly while we are on the subject, what was the reason for introducing this `Async.eachLimit` here when sending the messages to the client? - I think the `client.emit` should always be non-blocking but something must have prompted adding this?

Reply:
I think we needed to add this because we introduced a call to `Utils.getClientAttributes` inside the loop, on each client: https://github.com/overleaf/real-time/pull/81/files#diff-6888826980b5c4524173b5fa6eff8ecbR97

Now that we're removing the `getClientAttributes` call, I think we can go back to a plain for-loop.

Reply:
Ah right, now I understand. That will be a nice cleanup to go back to a simple loop @das7pad.

Reply:
I filed a new PR which is targeting this PRs' branch: https://github.com/overleaf/real-time/pull/115